test/compilable/b1215.d

```diff
@@ -0,0 +1,118 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS:
+
+/*
+TEST_OUTPUT:
+---
+TEST: base use case
+TEST: chained types
+TEST: chained packs
+TEST: expr
+TEST: Nested + index eval
+TEST: index with constexpr
+TEST: Nested + index with constexpr
+TEST: alias, base use case
+TEST: alias, chained types
+TEST: alias, chained packs
+TEST: alias, expr
+TEST: alias, Nested + index eval
+TEST: alias, index with constexpr
+TEST: alias, Nested + index with constexpr
+---
+*/
+
+import std.stdio;
+
+struct A(Args...)
+{
+    enum i = 1;
+
+    pragma(msg, "TEST: base use case");
+    Args[0].T mBase;
```

Comment:
To test the parsing and semantic analysis result, you can directly test the variable type, aliased type, and aliased value like follows:

```d
Args[0].T mBase;
static assert(is(typeof(mBase) == B.T));

alias UChain = Args[0].T.TT;
static assert(UChain == B.T.TT));

alias uExpr = Args[1].i;
static assert(uExpr == B.i);
```